Core.DomainModel/IHasRightsHolder.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Infrastructure.Services.Types;
+
+namespace Core.DomainModel
+{
+    public interface IHasRightsHolder
+    {
+        Maybe<int> GetRightsHolderOrganizationId();
```

Comment:
Denne refererer vel til objektets rettighedshaver organisations Id? Som i at det er organisation X som "ejer" It systemet og ikke bruger Y som "ejer" det.

Reply:
ja det står der også i metodenavnet eller?

Reply:
@JacobMalling ☝️